tslib/src/functions-throttled/ps-report-2-pupil-data/models.ts

```diff
@@ -0,0 +1,129 @@
+import moment from 'moment'
+
+export type nullOrUndef = undefined | null
+
+export interface Pupil {
+  attendanceId: number | undefined
+  checkComplete: boolean
+  currentCheckId: number | undefined
+  dateOfBirth: moment.Moment
+  forename: string
+  gender: 'M' | 'F'
+  id: number
+  lastname: string
+  notTakingCheckReason: string | undefined
+  slug: string
+  schoolId: number
+  upn: string
+}
+
+export interface School {
+  estabCode: number
+  id: number
+  laCode: number
+  name: string
+  slug: string
+  urn: number
+}
+
+export interface CheckConfig {
+  audibleSounds: boolean
+  checkTime: number
+  colourContrast: boolean
+  fontSize: boolean
+  inputAssistance: boolean
+  loadingTime: number
+  nextBetweenQuestions: boolean
+  numpadRemoval: boolean
+  practice: boolean
+  questionReader: boolean
+  questionTime: number
+  compressCompletedCheck: boolean
+}
+
+// interface CheckWindow {
```

Comment:
how come these are commented?

Reply:
They will be uncommented when this draft PR is complete.